CRM/Admin/Form/Setting/Debugging.php

```diff
@@ -54,4 +55,17 @@ public function buildQuickForm() {
     parent::buildQuickForm();
   }
 
+  /**
+   * Process the form submission.
+   */
+  public function postProcess() {
+    $params = $this->controller->exportValues($this->_name);
+
+    if ($params['isProductionEnvironment'] === FALSE) {
+      Civi::settings()->set('mailing_backend', array('outBound_option' => CRM_Mailing_Config::OUTBOUND_OPTION_DISABLED));
+      CRM_Core_Session::setStatus(ts('Outbound emails have been disabled. Scheduled jobs have been prevented from being executed.'), ts("Non-production environment set"), "success");
```

Comment:
Please change message to ''Outbound emails have been disabled. Scheduled jobs will not run unless runInNonProductionEnvironment=TRUE is added as a parameter for a specific job"